WebService/Runtime/Config.cs

```diff
@@ -87,6 +96,18 @@ public Config(IConfigData configData)
             this.ServicesConfig = GetServicesConfig(configData);
             this.ClientAuthConfig = GetClientAuthConfig(configData);
             this.RateLimitingConfig = GetRateLimitingConfig(configData);
+            this.DeploymentConfig = GetDeploymentConfig(configData);
+        }
+
+        private static IDeploymentConfig GetDeploymentConfig(IConfigData configData)
+        {
+            return new DeploymentConfig
+            {
+                AzureSubscriptionDomain = configData.GetString(AZURE_SUBSCRIPTION_DOMAIN, "undefined.onmicrosoft.com"),
```

Comment:
Is this different just to show it's a domain or is it a required default value?

Reply:
not a required value, just a default non-valid one